Contribute to tableQA (#398)
huggingface/hub-docs: tasks/src/table-question-answering/about.md

## Use Cases

### SQL execution

> You can use Table Question Answering model to simulate SQL execution by using provided table.

Reviewer: ```suggestion
You can use the Table Question Answering models to simulate SQL execution by inputting a table.
```

Reviewer: Can you elaborate this a bit more, what do you input and what do you get after the inference?

### Table Question Answering 

> Given the question and the table, the task is to answer the question based on the table, You can use Table Question Answering model to answer question based on the table. 

Reviewer: ```suggestion
Table Question Answering models are capable of answering questions based on a table.
```
It was a bit long and ambiguous.

## Inference 

> You can infer with TableQA models with the 🤗 Transformers library.

Reviewer: ```suggestion
You can infer with TableQA models using the 🤗 Transformers library.
```

'''python 
from transformers import AutoTokenizer, AutoModelForSeq2SeqLM
import pandas as pd

tokenizer = AutoTokenizer.from_pretrained("microsoft/tapex-large-finetuned-wtq")
model = AutoModelForSeq2SeqLM.from_pretrained("microsoft/tapex-large-finetuned-wtq")

# prepare table + question
data = {"Actors": ["Brad Pitt", "Leonardo Di Caprio", "George Clooney"], "Number of movies": ["87", "53", "69"]}
table = pd.DataFrame.from_dict(data)
question = "how many movies does Leonardo Di Caprio have?"

encoding = tokenizer(table, question, return_tensors="pt")

> # let the model generate an answer autoregressively

Reviewer: We try to avoid using terms to have a broader audience 🙂
